src/main/java/com/thangle/api/book/BookDTOMapper.java

```diff
@@ -13,6 +13,25 @@ public static BookDTO toBookDTO(final Book book) {
         return BookDTO.builder()
                 .id(book.getId())
                 .title(book.getTitle())
+                .author(book.getAuthor())
+                .description(book.getDescription())
+                .createdAt(book.getCreatedAt())
+                .updatedAt(book.getUpdatedAt())
+                .image(book.getImage())
+                .userId(book.getUserId())
+                .build();
+    }
+
+    public static Book toBook(final BookDTO bookDTO) {
+        return Book.builder()
+                .id(bookDTO.getId())
```

Comment:
We don't use this value because
- When create a new book --> we don't have it
- When update a book --> we use id from path